### ISSUE TITLE
3.4.1 Example Module Using YANG Semver shows non-SemVer compliant version 1.1

### ISSUE THREAD

**Issue by jwdonahue** [netmod-wg/yang-ver-dt#86]
3.4.1 Example Module Using YANG Semver shows non-SemVer compliant version 1.1.

All SemVer strings MUST have the full version triple, or it's not SemVer.

```
3.4.1.  Example Module Using YANG Semver

   Below is a sample YANG module that uses the YANG semver revision
   label based on the rules defined in this document.

       module example-versioned-module {
         yang-version 1.1;
         namespace "urn:example:versioned:module";
         prefix "exvermod";
         rev:revision-label-scheme "yangver:yang-semver";
```

But this is YANG SemVer, so I suppose you can just do whatever you want, but then you should make it clear that your scheme is only tenuously tied to, or simply inspired by [semver.org](https://semver.org/).

**Comment by rgwilton** [comment]
This isn't an issue.

YANG semver is being used to version the content of the data module, not the YANG language itself.  I.e., you'll see the YANG Semver versions further down the example module.

I.e. the two versions of the YANG language are 1.0 (RFC 6020) and 1.1 (RFC 7950).  They are not strictly versioned using Semver, but arguably follow the same scheme, i.e. 1.1 is meant to be language backwards compatible to 1.0.

**Comment by schoenw** [comment]
The truth is in section 1.1 of RFC 7950: While most YANG 1 modules (yes it was YANG 1 not YANG 1.0) continue to work without changes as YANG 1.1 modules, there are corner cases where this is not true. People wanted stability and hence the YANG revision project was started with the goal to produce YANG 1.1, hence we tried to maximize compatibility with YANG 1 but there are some cases where we did have to accept potential incompatibilities. For me, version numbers are always the result of a judgement process, and this is where my views likely differ substantially from true SemVer enthusiasts.
